util/caller/resolver.go

```diff
@@ -49,18 +49,30 @@ type CallResolver struct {
 var defaultRE = func() *regexp.Regexp {
 	_, file, _, ok := runtime.Caller(0)
 	if !ok {
-		panic("unable to look up location")
+		return regexp.MustCompile(`^$`) // strip nothing
 	}
 	const sep = string(os.PathSeparator)
+	qSep := regexp.QuoteMeta(sep)
 	path := filepath.Dir(file)
+	// Coverage tests report back as `[...]/util/caller/_test/_obj_test`;
+	// strip back to this package's directory.
+	for strings.Contains(path, sep) && !strings.HasSuffix(path, "caller") {
```

Comment:
doesn't this stop one iteration too soon? seems like you'll get `[...]/util/caller` rather than `[...]/util`.